src/flashbar/__tests__/flashbar.test.tsx

```diff
@@ -675,4 +675,9 @@ describe('Analytics', () => {
       })
     );
   });
+
+  test('adds the correct data-analytics tag', () => {
```

Comment:
Since there's some extra logic involved with determining the "`effectiveType`", I think it's worth adding a couple more tests here.